scriptmodules/supplementary/xboxdrv.sh

```diff
@@ -31,7 +31,18 @@ function install_xboxdrv() {
 }
 
 function enable_xboxdrv() {
-    local config="\"$md_inst/bin/xboxdrv\" --daemon --detach --dbus disabled --detach-kernel-driver --id 0 --led 2 --deadzone 4000 --silent --trigger-as-button --next-controller --id 1 --led 3 --deadzone 4000 --silent --trigger-as-button"
+    if [[ -n "$controllers_wanted" ]]; then
+
+	local config="\"$md_inst/bin/xboxdrv\" --daemon --detach --dbus disabled --detach-kernel-driver --id 0 --led 2 --deadzone $deadzone_wanted --silent --trigger-as-button"
+        local loop="1"
+
+	while [ "$loop" -lt "$controllers_wanted" ]
```

Comment:
doesn't actually metter - just mentioning it. please use double brackets though as per our styleguide https://github.com/RetroPie/RetroPie-Setup/wiki/Shell-Style-Guide